README.md

```diff
@@ -128,7 +131,8 @@ module.exports = {
       allowEmptyValues: true, // allow empty variables (e.g. `FOO=`) (treat it as empty string, rather than missing)
       systemvars: true, // load all the predefined 'process.env' variables which will trump anything local per dotenv specs.
       silent: true, // hide any errors
-      defaults: false // load '.env.defaults' as the default values if empty.
+      defaults: false, // load '.env.defaults' as the default values if empty.
+      prefix: 'import.meta.env' // reference your env variables as 'import.meta.env.ENV_VAR'.
```

Comment:
All of these examples here are the defaults. I would adjust yours to show “process.env.” to maintain consistency.

Reply:
Hello! As I see, the defaults are set only for `path`, all other options are undefined and coerced to false in conditions. That's why I assumed putting here possible option instead of default. Am I missing something here?
https://github.com/mrsteele/dotenv-webpack/blob/bf50d8d9b6436357e183a1ff285427e387cb5b35/src/index.js#L33-L35

Reply:
Oh wow, you are right I’m not sure where my head was at. You can disregard this bit.